usb/usb_host_msc/test/msc_device.c

```diff
@@ -1,92 +1,155 @@
 /*
```

Comment:
General comment to this file:
The main idea of refactoring this MSC mock device, was to get rid of the TinyUSB callbacks (such as `tud_msc_test_unit_ready_cb`, `tud_msc_capacity_cb`, `tud_***_cb`...) and use the callbacks implementation from `esp_tinyusb`.

The goal is to showcase that implementing a MSC device with `esp_tinyusb` is easy -> that the user does _not_ have to bother with defining all these TinyUSB callbacks. Could you please take a look whether we can remove these callback from this file?

Reply:
tinyusb callbacks are defined in file msc_test.c for idf v4.4.
idf v4.4 dos not use esp_tinyusb (spiflash and sdcard), So, we cannot remove tinyusb callbacks definition from TC file msc_device.c